### PR TITLE
ENH: Launch model asynchronously

### PR DESCRIPTION
**Pull request by ChengjieLi28**

### CODE REVIEW

File: xinference/deploy/local.py [hunk 1]
```diff
@@ -46,6 +47,9 @@ async def _start_local_cluster(
         pool = await create_worker_actor_pool(
             address=address, logging_conf=logging_conf
         )
+        await xo.create_actor(
```
Comment (aresnow1) [inline xinference/deploy/local.py:50]: Better to create the actor in `SupervisorActor.__post_create__`.

**Review by aresnow1 — COMMENTED** [review]
Better add a test

**Comment by ChengjieLi28** [comment]
> Better add a test

Added.

**Review by aresnow1 — APPROVED** [review]
LGTM

**Pull request merged**